### PR TITLE
Create PR-Automation.yml

### PR DESCRIPTION
**Pull request by zdmullen**
## Proposed changes

This will introduce automation into our GitHub Project.

## Release Notes

Milestone:

Changelog:

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] `npm run vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: .github/workflows/PR-Automation.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  pull_request:
+    types: [opened]
+
+jobs:
+  automate-project-columns:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: alex-page/github-project-automation-plus@v0.2.4
+        with:
+          project: test
```
Comment (fox0r) [inline .github/workflows/PR-Automation.yml:13]: change test to Explorers' Squad

File: .github/workflows/PR-Automation.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+          column: In progress
```
Comment (fox0r) [inline .github/workflows/PR-Automation.yml:14]: change In progress to In dev

File: .github/workflows/PR-Automation.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+name: Move assigned pull requests into To do
```
Comment (fox0r) [inline .github/workflows/PR-Automation.yml:1]: change to Move opened pull request In Dev

**Comment by zdmullen** [comment]
Does this look better now, @fox0r ?

**Comment by fox0r** [comment]
> Does this look better now, @fox0r ?

I approve this beautiful masterpiece  :)

**Comment by dkelosky** [comment]
I know we discussed getting a description of what this does.  Perhaps a different name could be set to help clarify the workflow's purpose:
image

**Comment by zFernand0** [comment]
> I know we discussed getting a description of what this does. Perhaps a different name could be set to help clarify the workflow's purpose:
> 
> image

@dkelosky, any suggestions?

**Comment by dkelosky** [comment]
I don't really understand what it does besides introduce automation.  @zdmullen can you clarify what this does and then we can pick a name to reflect that?

**Comment by katelynienaber** [comment]
@zdmullen Whatever happened with this PR? Is it ever going to be merged?

**Comment by zFernand0** [comment]
> @zdmullen Whatever happened with this PR? Is it ever going to be merged?

I still like the idea of using GH Projects to better manage the Monorepo. :yum:

**Comment by jellypuno** [comment]
@zdmullen Can we close this PR? Are there still plans to implement this?

**Comment by jellypuno** [comment]
No response received. Closing this PR...

**Pull request closed without merging**